pkg/yurtctl/cmd/revert/revert.go

```diff
@@ -194,15 +195,25 @@ func (ro *RevertOptions) RunRevert() (err error) {
 		return
 	}
 
-	// 6. recreate the node-controller service account
-	ncSa := &v1.ServiceAccount{
+	// 6. recreate the system:controller:node-controller clustrrolebinding
+	ncClusterrolebinding := &rbacv1.ClusterRoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "node-controller",
-			Namespace: "kube-system",
+			Name: "system:controller:node-controller",
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     "system:controller:node-controller",
+		},
+		Subjects: []rbacv1.Subject{
+			rbacv1.Subject{
+				Kind:      "ServiceAccount",
+				Name:      "node-controller",
+				Namespace: "kube-system",
+			},
 		},
 	}
-	if _, err = ro.clientSet.CoreV1().
-		ServiceAccounts(ncSa.GetNamespace()).Create(ncSa); err != nil && !apierrors.IsAlreadyExists(err) {
+	if _, err = ro.clientSet.RbacV1().ClusterRoleBindings().Create(ncClusterrolebinding); err != nil && !apierrors.IsAlreadyExists(err) {
 		klog.Errorf("fail to create node-controller service account: %s", err)
```

Comment:
how about update the log of service account to clusterrolebinding

Reply:
fixed